lib/shopify_api/graphql.ex

```diff
@@ -60,17 +60,26 @@ defmodule ShopifyAPI.GraphQL do
     actual_cost =
       metadata
       |> get_in(["cost", "actualQueryCost"])
-      |> Kernel.trunc()
+      |> case do
```

Comment:
These were modified as tests were failing when log level was turned down to `debug` in test environment.